src/collective/cover/static/layout_edit.js

```diff
@@ -113,11 +113,14 @@
              **/
             get_random_value: function(c) {
               var r;
-              if (typeof crypto !== "undefined" && crypto !== null) {
+              if (window.crypto && typeof window.crypto.getRandomValues === 'function') {
                 r = crypto.getRandomValues(new Uint8Array(1))[0] % 16 | 0;
               } else { // Fallback for older browsers
                 r = Math.random() * 16 | 0;
               }
+              // Improve randomness. See: http://stackoverflow.com/a/8809472/2116850
+              r = (self.timestamp + r) % 16 | 0;
+              self.timestamp = Math.floor(self.timestamp / 16);
```

Comment:
Mix timestamp with our random number in the first line, and update timestamp variable to get next portion of the number in the next loop cycle.